worker/include/RTC/RtpReceiver.h

```diff
@@ -56,6 +56,9 @@ namespace RTC
 	private:
 		void FillRtpParameters();
 
+	private:
+		static uint8_t rtcpBuffer[];
```

Comment:
Same as above: let's move `static` members declaration to the top of the class, before instance members.